enhancements/console/dynamic-plugins.md

```diff
@@ -433,6 +433,34 @@ const VMHeading = () => {
 };
 ```
 
+In the 4.11 release a `console.openshift.io/default-i18next-namespace` annotation,
```

Comment:
I would move this below discussion of the API property we're introducing since it's not the recommended way and only here for backwards compatibility.